src/frame.scaler.hh

```diff
@@ -0,0 +1,60 @@
+#ifndef H_ACQUIRE_ZARR_FRAME_SCALER_V0
+#define H_ACQUIRE_ZARR_FRAME_SCALER_V0
+
+#ifdef __cplusplus
+
+#include <condition_variable>
+#include <mutex>
+#include <optional>
+#include <queue>
+#include <unordered_set>
+
+#include "prelude.h"
+#include "tiled.frame.hh"
+#include "chunk.writer.hh"
+
+namespace acquire::sink::zarr {
+class Zarr;
+
+struct Multiscale
```

Comment:
`Multiscale` is a bit confusing to me. Maybe `ScalingParameters` or something.  If it's primarily used by the `FrameScaler` maybe something like `FrameScalerConfig` to indicate where it's used.

Reply:
It's used in the Zarr object. I'll rename it to `ScalingParameters`.